.htaccess

```diff
@@ -35,9 +35,9 @@ FileETag MTime Size
 	RewriteRule src/Backend/Cache/Locale/.*\.php - [F]
 
 	# redirect all trafic to https
-	# RewriteCond %{SERVER_PORT} 80
+	# RewriteCond %{HTTPS} != on
 	# RewriteCond %{HTTP_HOST} !.*\.dev [NC]
-	# RewriteRule ^(.*)$ https://www.<domain>.be/$1 [R=301,L]
+	# RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]
```

Comment:
Better comment it out too. And enable it when enabling https

> **The only secure way to redirect http to https is to use HSTS (Header Strict-Transport-Security) .** The apache redirect is insecure because an attacker can intercept it and rewrite it. Unfortunately, for older browser and browser how didn't preload HSTS, it's your only option. HSTS is the only reliable protection against SSLTrip
> It tells the browsers to only use https on that domain and all sub-domains for one year (all http request will be rewrite as https request without network interaction)

> HTTP Strict Transport Security (HSTS) is an opt-in security enhancement that is specified by a web application through the use of a special response header. Once a supported browser receives this header that browser will prevent any communications from being sent over HTTP to the specified domain and will instead send all communications over HTTPS. It also prevents HTTPS click through prompts on browsers.
> https://www.owasp.org/index.php/HTTP_Strict_Transport_Security_Cheat_Sheet